app/views/components/_figure.html.erb

```diff
@@ -1,12 +1,21 @@
 <%
   alt ||= ''
   caption ||= ''
+  credit ||= ''
 %>
 <figure class="app-c-figure">
   <img class="app-c-figure__image" src="<%= src %>" alt="<%= alt %>">
-  <% unless caption.blank? %>
     <figcaption class="app-c-figure__figcaption">
-      <%= caption %>
+      <% unless caption.blank? %>
+        <div class="app-c-figure__figcaption-text">
+          <%= caption %>
+        </div>
+      <% end %>
+
+      <% if credit.present? %>
+          <div class="app-c-figure__figcaption-credit">
+            Image credit: <%= credit %>
```

Comment:
Should "Image credit" come from a translation file?

Reply:
Do you know what happens if there isn't a translation available for the other languages?

Reply:
I thought it fell back to the english one... but you can provide a default too?

Reply:
You're right.

> The I18n library will use English as a default locale, i.e. if a different locale is not set, :en will be used for looking up translations.

From: https://guides.rubyonrails.org/i18n.html

Reply:
@vanitabarrett Updated and fixed up.